### ISSUE TITLE
Doesn't compile

### ISSUE THREAD

**Issue by a1b0hph0b1a** [weedo3d/Tina2Upgradefirmware#3]
Code doesn't seem to compile...  I haven't changed anything yet, installed the PlatformIO maple profile, but doesn't compile with following error...

RuntimeError: deque mutated during iteration:
  File "C:\Users\total\.platformio\penv\lib\site-packages\platformio\builder\main.py", line 181:
    env.SConscript(env.GetExtraScripts("post"), exports="env")
  File "C:\Users\total\.platformio\packages\tool-scons\scons-local-4.5.2\SCons\Script\SConscript.py", line 598:
    return _SConscript(self.fs, *files, **subst_kw)
  File "C:\Users\total\.platformio\packages\tool-scons\scons-local-4.5.2\SCons\Script\SConscript.py", line 285:
    exec(compile(scriptdata, scriptname, 'exec'), call_stack[-1].globals)
  File "D:\Printer Firmwares\tina2\Tina2Upgradefirmware-main\buildroot\share\PlatformIO\scripts\WEEDOR72.py", line 5:
    for define in env['CPPDEFINES']:

**Comment by ngoels** [comment]
I have the same problem - is there any solution?

**Comment by ngoels** [comment]
Fixed it for me:
in /buildroot/share/PlatformIO/scripts/WEEDOR72.py (line 4)

```
# Relocate firmware from 0x08000000 to 0x08008000
#for define in env['CPPDEFINES']:   
envdefs = env['CPPDEFINES'].copy()
for define in envdefs:
```


I found this one related to it: https://github.com/MarlinFirmware/Marlin/commit/c2decc3e2e30c7cb0f517b7e40d8138a8c1d4a81

**Comment by a1b0hph0b1a** [comment]
> Fixed it for me: in /buildroot/share/PlatformIO/scripts/WEEDOR72.py (line 4)
> 
> ```
> # Relocate firmware from 0x08000000 to 0x08008000
> #for define in env['CPPDEFINES']:   
> envdefs = env['CPPDEFINES'].copy()
> for define in envdefs:
> ```
> 
> I found this one related to it: [MarlinFirmware/Marlin@c2decc3](https://github.com/MarlinFirmware/Marlin/commit/c2decc3e2e30c7cb0f517b7e40d8138a8c1d4a81)

That did actually work. Have you tried flashing it to the printer yet?

PS - Now M73 doesn't work! :'(

**Comment by ngoels** [comment]
> > Fixed it for me: in /buildroot/share/PlatformIO/scripts/WEEDOR72.py (line 4)
> 
> > 
> 
> > ```
> 
> > # Relocate firmware from 0x08000000 to 0x08008000
> 
> > #for define in env['CPPDEFINES']:   
> 
> > envdefs = env['CPPDEFINES'].copy()
> 
> > for define in envdefs:
> 
> > ```
> 
> > 
> 
> > I found this one related to it: [MarlinFirmware/Marlin@c2decc3](https://github.com/MarlinFirmware/Marlin/commit/c2decc3e2e30c7cb0f517b7e40d8138a8c1d4a81)
> 
> 
> 
> That did actually work. Have you tried flashing it to the printer yet?
> 
> 
> 
> PS - Now M73 doesn't work! :'(

Yes, it compiles and I was able to flash it without any other problems.
